### PR TITLE
APP-2283 Migrate Breadcrumbs to @viamrobotics/prime-core

### PR DESCRIPTION
**Pull request by ethanlook**
Migrates Breadcrumbs component to `@viamrobotics/prime-core`.

## Change log

- Copy `breadcrumbs.svelte` to `packages/core`
    - Change props to accept an array
- Add `Breadcrumbs` export from `index.ts`
- Copy unit tests
    - Change to use vitest
- Copy storybook story to `packages/storybook`
    - Change to use Svelte component directly

## Testing

- Added unit tests

## Review requests

- None

### CODE REVIEW

File: packages/core/src/lib/breadcrumbs.spec.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+import { render, screen } from '@testing-library/svelte';
+import Breadcrumbs from './breadcrumbs.svelte';
+
+describe('Breadcrumbs', () => {
+  it('Renders breadcrumbs with the list of values specified in crumbs attribute as pills', () => {
+    render(Breadcrumbs, { crumbs: ['Chocolate Chip', 'Oatmeal Raisin'] });
+    expect(screen.getByText('Chocolate Chip')).toBeInTheDocument();
+    expect(screen.getByText('Oatmeal Raisin')).toBeInTheDocument();
+  });
+});
```
Comment (ethanlook) [inline packages/core/src/lib/breadcrumbs.spec.ts:11]: Add a test for empty crumbs array

Comment (ethanlook) [reply packages/core/src/lib/breadcrumbs.spec.ts:11]: No longer necessary since the crumbs prop is now `[string, ...string[]]`

**Review by mcous — APPROVED** [review]
Lgtm

File: packages/core/src/routes/+page.svelte [hunk 2]
```diff
@@ -23,3 +24,6 @@ import Badge from '$lib/badge.svelte';
   variant="blue"
   label="Info"
 />
+
+<!-- Breadcrumbs -->
+<Breadcrumbs crumbs={['Chocolate Chip', 'Oatmeal Raisin']} />
```
Comment (micheal-parks) [inline packages/core/src/routes/+page.svelte:29]: Maybe we should start splitting this into individual routes?

**Review by micheal-parks — APPROVED** [review]

File: packages/core/src/routes/+page.svelte (hunk 2 above)
Comment (ethanlook) [reply packages/core/src/routes/+page.svelte:29]: I could go either way, let's start here and split once it becomes unruly?
